### PR TITLE
support for just .spikes in folder

### PR DESCRIPTION
**Pull request by ferchaure**
Hi I was trying to load an openephys _.spikes_ file but it required the .continuous file as well. To fix that, I separated a bit better the data loading, all the code was there. 
And I fixed a typo when the filename is read to check the number of segments.

This will work for my case, but probably a similar fix could be made to load only events files.

Cheers,
Fer

### CODE REVIEW

**Comment by ferchaure** [comment]
Maybe the error is related to a unreported renaming in my data, I will check it that and fix it

**Comment by samuelgarcia** [comment]
Hi fernando.
Thank you. this is Ok for me.
Can you fix the warning line 495 ?

File: neo/rawio/openephysrawio.py [hunk 1]
```diff
@@ -485,11 +492,11 @@ def explore_folder(dirname):
             if (seg_index + 1) > info['nb_segment']:
                 info['nb_segment'] += 1
         elif filename.endswith('.spikes'):
-            s = filename.replace('.spikes', '').split('_')
-            if len(s) == 1:
-                seg_index = 0
+            s = re.findall("(_\d+)$", filename.replace('.spikes', ''))
```
Comment (samuelgarcia) [inline neo/rawio/openephysrawio.py:495]: pep8 give a warning. I guess maybe you have to do \\ no ?
can you check ?

Comment (JuliaSprenger) [reply neo/rawio/openephysrawio.py:495]: @samuelgarcia Feel free commit the suggestion and see if it resolves the pep8 issue.

**Comment by alejoe91** [comment]
Closing in favor of #1377

**Pull request closed without merging**